src/scripts/main.js

```diff
@@ -2,6 +2,17 @@
 
 const pushNotification = (posTop, posRight, title, description, type) => {
   // write code here
+  const mess = document.createElement('div');
+
+  mess.style.top = `${posTop}px`;
+  mess.style.right = `${posRight}px`;
+  mess.innerHTML = `<h2 class='title'>${title}</h2><p>${description}</p>`;
+  mess.classList.add('notification', type);
+  document.body.append(mess);
+
+  setTimeout(() => {
+    mess.style.display = 'none';
```

Comment:
Instead of setting `mess.style.display = 'none';`, consider using `mess.remove();` to completely remove the element from the DOM. This will help in managing memory and performance, especially if many notifications are created over time.